File: snippets/api/source-lines/player-set.java
PlasmoVoiceServer voiceServer = /* */;

ServerSourceLine sourceLine = voiceServer.getSourceLineManager()
        .createBuilder(/*...*/)
        .withPlayers(true)
        .build();

VoicePlayer player = /* voice player */;

ServerPlayerSetManager playerSetManager = sourceLine.getPlayerSetManager();
ServerPlayerSet broadcastSet = playerSetManager.createBroadcastSet();
broadcastSet.addPlayer(player);

// sets the broadcast player set to the player
playerSetManager.setPlayerSet(player, broadcastSet);

File: snippets/api/source-lines/player-set.kt
val voiceServer: PlasmoVoiceServer

val sourceLine: ServerSourceLine = voiceServer.sourceLineManager
    .createBuilder(/* */)
    .withPlayers(true)
    .build()

val player: VoicePlayer /* voice player */

val playerSetManager = sourceLine.playerSetManager
val broadcastSet = playerSetManager.createBroadcastSet()
broadcastSet.addPlayer(player)

// sets the broadcast player set to the player
playerSetManager.setPlayerSet(player, broadcastSet)
